File: C/1012.c
#include <stdio.h>
#include <math.h>
#define pi 3.14159
int main()
{
    double A,B,C,tri,cir,tra,sq,rec;
    scanf("%lf %lf %lf",&A,&B,&C);
    tri=0.5*A*C, cir=pi*C*C, tra=0.5*(A+B)*C, sq=B*B, rec=A*B;
    printf("TRIANGULO: %.3lf\nCIRCULO: %.3lf\nTRAPEZIO: %.3lf\nQUADRADO: %.3lf\nRETANGULO: %.3lf\n",tri,cir,tra,sq,rec);
    return 0;
}

File: Python/1012.py
PI=3.14159
a,b,c=[float(x) for x in input().split()]
tri,cir,tra,sq,rec=0.5*a*c,PI*pow(c,2),0.5*(a+b)*c,pow(b,2),a*b
print(f"TRIANGULO: {tri:.3f}\nCIRCULO: {cir:.3f}\nTRAPEZIO: {tra:.3f}\nQUADRADO: {sq:.3f}\nRETANGULO: {rec:.3f}")